Sources/Views/Cells/MessageCollectionViewCell.swift

```diff
@@ -59,114 +53,80 @@ open class MessageCollectionViewCell<ContentView: UIView>: UICollectionViewCell,
 
     open weak var delegate: MessageCellDelegate?
 
-    // MARK: - Initializer
-
-    public override init(frame: CGRect) {
+    override public init(frame: CGRect) {
         super.init(frame: frame)
+        contentView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
```

Comment:
Is this `autoresizingMask` necessary? Cell would manage `contentView`'s frame in my opinion.

Reply:
I'm going to keep it for now because I can't remember why it is here 😅 I think there may have been a small problem on rotation